### PR TITLE
Relax the int vs float constraint

### PR DESCRIPTION
**Pull request by gbin**
As in JSON encoding they are equivalent, we can let them through.

### CODE REVIEW

File: apitools/base/protorpclite/messages.py [hunk 1]
```diff
@@ -1321,7 +1321,9 @@ def validate_element(self, value):
         Raises:
           ValidationError if value is not expected type.
         """
-        if not isinstance(value, self.type):
+        if not (isinstance(value, self.type) or
+            (isinstance(value, (float, int)) and
```
Comment (craigcitro) [inline apitools/base/protorpclite/messages.py]: this test is looking a little hard to read -- can we just have

    if isinstance(value, int) and self.type == float:
      return

as the first few lines inside the `if`?

(note that i'm also trimming the logic to just what we need for now -- float -> int conversion is more worrisome, due to precision/rounding questions, i think?)

**Comment by gbin** [comment]
PTAL

**Comment by craigcitro** [comment]
LGTM, but the linter wants you to fix a long line (in messages_test). tweak that and i'll merge.

thanks for cleaning this up!

**Comment by gbin** [comment]
PTAL.

**Pull request merged**